example/lib/main.dart

```diff
@@ -24,6 +24,10 @@ class _MyAppState extends State<MyApp> {
   String _recorderTxt = '00:00:00';
   String _playerTxt = '00:00:00';
 
+  double slider_current_position = 0.0;
+  double max_duration = 0.0;
```

Comment:
The initial duration should be greater than `0` so I stated to `1.0` for initial state.